### ISSUE TITLE
Double-quotation mark as a type-level char literal breaks highlighting

### ISSUE THREAD

**Issue by kirelagin** [atom-haskell/language-haskell#128]
“The Char Kind” proposal (https://github.com/ghc-proposals/ghc-proposals/pull/387) has been recently accepted and [its implementation](https://gitlab.haskell.org/ghc/ghc/-/merge_requests/4351) will ship in GHC 9.2.1.

Here is the [T11342a test](https://gitlab.haskell.org/ghc/ghc/-/blob/9c9e40e59214b1e358c85852218f3a67e712a748/testsuite/tests/typecheck/T11342/T11342a.hs) with all its char-literals replaced with `"`:

```haskell
{-# LANGUAGE DataKinds #-}
{-# LANGUAGE KindSignatures #-}
{-# LANGUAGE TypeOperators #-}

module T11342a where

import Data.Type.Equality

type A = '"' :: Char

-- I am a blue comment

t :: '"' :~: '"'
t = Refl
```

![image](https://user-images.githubusercontent.com/451835/112769755-a9492c00-8ff0-11eb-960f-37926358948a.png)

**Comment by kirelagin** [comment]
(Just to be clear: I am not a user of Atom, this is an issue with highlighting on github.com, but I am reporting it here, since I read somewhere on the internet that github.com uses this code for highlighting.)

**Comment by lierdakil** [comment]
Thanks for reporting. Fixed and released as v1.22.0. GitHub will (eventually) pick this up, usually in 2 weeks to a month.
